test/specs/elements/Button/Button-test.js

```diff
@@ -109,6 +109,11 @@ describe('Button', () => {
       wrapper.find('Label').should.not.have.className('disabled')
       wrapper.find('button').should.not.have.className('disabled')
     })
+    it('contains children without floated class when floated attribute is set', () => {
```

Comment:
Would prefer we assert the expected location of the floated class in this test as well. It currently asserts where the className should _not_ be, but doesn't assert where it _should_ be.